### DISCUSSION TITLE
Size of community vs market share

### DISCUSSION THREAD

**Discussion by ryansunxl** [saleor/saleor#13067] [General] [answered]
I strongly believe community is the fuel of a opensource project and will drive the adoption. In the end of the day, it is the technical professions, which may well be part of the community or been influenced by the community, making suggestions to ecommerce platforms. 

Github stars wise, saleor surpasses all the other opensource ecommerce projects (saleor 18.6k, magento 10.7k, woocommerce 8.6k, spree 12.3k, shopware 1.8k), but saleor is not even on the marketshare report https://trends.builtwith.com/shop.

Anyone any thoughts on this? 
Is what I believed wrong (community size =/= adoption rate)? Or something about saleor making it not best fit for business?

**Comment by patrys** [comment, marked as answer]
I can explain why that's the case. Because of security considerations, we do not provide any way for outsiders to identify Saleor or its precise version. This is to prevent automatic scanners from finding vulnerable sites each time we publish a CVE, a known problem for software like WordPress and WooCommerce. An unfortunate side effect is that legitimate stack analyzers like BuiltWith cannot reliably detect Saleor being part of the solution.
